Resources/Prototypes/Actions/station_ai.yml

```diff
@@ -13,24 +14,37 @@
 
 - type: entity
   id: ActionShowJobIcons
-  name: Show job icons
-  description: Shows job icons for crew members.
+  name: Show status icons
+  description: Shows status icons for crew members.
   components:
   - type: InstantAction
+    priority: -5
     itemIconStyle: BigAction
     icon:
       sprite: Interface/Actions/actions_ai.rsi
       state: job_view
     event: !type:ActionComponentChangeEvent
       components:
       - type: ShowJobIcons
+      - type: ShowMindShieldIcons
+      - type: ShowCriminalRecordIcons
+      #TODO: Add those in when theres no desync issue with ActionComponentChangeEvent
+      #- type: ShowHealthBars
+       # damageContainers:
+        #- Biological
+        #- Inorganic
+        #- Silicon
+      #- type: ShowHealthIcons
+       # damageContainers:
+        #- Biological
```

Comment:
These are not mentioned in the PR description at all. Make that a separate PR. and don't include commented out code for features that haven't been decided on yet.